pcsx2/gui/MemoryCardFile.cpp

```diff
@@ -359,7 +363,23 @@ s32 FileMemoryCard::Save( uint slot, const u8 *src, u32 adr, int size )
 	}
 
 	if( !Seek(mcfp, adr) ) return 0;
-	return mcfp.Write( m_currentdata.GetPtr(), size ) != 0;
+
+	int status = mcfp.Write( m_currentdata.GetPtr(), size );
+
+	if( 0 != status ) {
```

Comment:
I don't mind either way.  It's just a defensive programming style to protect against accidentally using an assignment instead of a comparison.  The compiler will throw an error on (const = var) because you can't assign a value to a const but it will happily accept (var = const), because sometimes you really want to do that in more complex statements like (var1 == (var2 = func())